README.md

```diff
@@ -401,8 +408,7 @@ Total: 100 / 100
 ```
 
 Extending the previous example, we can manually set the time of submission through `proxy-time`.
-We can use a [Unix Epoch time converter](https://www.epochconverter.com/) to change the target time
-of March, 4th 2025 at 12:00 PM Pacific Time to milliseconds from the Unix Epoch, `1741118400000`:
+We will use the Unix Epoch time `1741118400000` that we converted above.
 ```sh
 python3 -m autograder.cli.courses.assignments.submissions.proxy.submit --proxy-email student@test.edulinq.org --proxy-time 1741118400000 my_file.py
```

Comment:
This is up to you, but you may not need to show the output for this command.
(Especially since there isn't anything special in it.)